packages/manager/modules/pci/src/projects/new/components/pci-new-project-promotion-banner/controller.js

```diff
@@ -0,0 +1 @@
+export default class PciNewProjectPromotionBannerController {}
```

Comment:
It isn't needed if is empty.

Reply:
it is needed because I need to access the `$ctrl` context in the template in order to get the component props.

Reply:
I think we can call $ctrl even if we don't have a controller. In the codebase I can't find a place where we have an empty controller.

Reply:
Removing the controller class as it is will make the `$ctrl` context inaccessible.

Reply:
Interesting point, i understand @brofik point and i know we don't need it if it's empty like @ghyenne said.
So, i guess we use parent one or somewhere something init an empty controller.
I am curios, from your component disable the controller and try to use this into `template.html`:
`<span>Ctrl.name:: {{$ctrl.constructor.name}}</span>`

 Let me know.